dynawo/sources/Models/Modelica/Dynawo/Examples/ENTSOE/TestCase1.mo

```diff
@@ -138,7 +138,7 @@ equation
+  Documentation(info = "<html><head></head><body><span style=\"left: 118.2px; top: 569.49px; font-family: sans-serif;\">The purpose of the first test case is to compare the dynamic behaviou</span><span style=\"left: 698.591px; top: 569.49px; font-family: sans-serif;\">r of the model for the </span><span style=\"left: 118.2px; top: 594.09px; font-family: sans-serif;\">synchronous </span><span style=\"left: 234.194px; top: 594.09px; font-family: sans-serif;\">machine </span><span style=\"left: 315.393px; top: 594.09px; font-family: sans-serif;\">and </span><span style=\"left: 357.989px; top: 594.09px; font-family: sans-serif;\">its    AVR</span><span style=\"left: 425.977px; top: 594.09px; font-family: sans-serif;\">  by </span><span style=\"left: 469.18px; top: 594.09px; font-family: sans-serif;\">analysing</span><span style=\"left: 546.773px; top: 594.09px; font-family: sans-serif;\">  the  terminal  voltage</span><span style=\"left: 794.4px; top: 594.09px; font-family: sans-serif;\">&nbsp;and  the  </span><span style=\"left: 118.2px; top: 622.886px; font-family: sans-serif;\">excitation  voltage </span><span style=\"left: 306.2px; top: 622.89px; font-family: sans-serif;\">inside</span><span style=\"left: 363.994px; top: 622.89px; font-family: sans-serif;\">  the </span><span style=\"left: 409.387px; top: 622.89px; font-family: sans-serif;\">machine</span><span style=\"left: 478.59px; top: 622.89px; font-family: sans-serif;\">.&nbsp;</span><div><span style=\"left: 478.59px; top: 622.89px; font-family: sans-serif;\">The test consists of a no-load operation with a step on the voltage reference value (+ 0.05 pu) done at t = 0.1 s.</span></div><div><span style=\"left: 478.59px; top: 622.89px; font-family: sans-serif;\"><br></span></div><div><span style=\"left: 478.59px; top: 622.89px; font-family: sans-serif;\">The results obtained perfectly match the results presented in the ENTSO-E report.</span></div>
```

Comment:
dynamic behaviour instead of dynamic behaviou

Reply:
Corrected